admin/qbee-agent/Makefile

```
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=qbee-agent
+PKG_VERSION:=2024.23
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_RELEASE:=1
+PKG_GIT_COMMIT:=3219c17730eab4c80598494732b4bed4c4c18b51
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_GIT_URL:=github.com/qbee-io/qbee-agent
+PKG_GIT_REF:=$(PKG_VERSION)
+PKG_SOURCE_URL:=https://$(PKG_GIT_URL)
+PKG_MIRROR_HASH:=52712fb50d471fd6e723689f021dab7cd51901bf30aee8aa85ec96fb5d639ae0
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MAINTAINER:=Jon Henrik Bjørnstad <jonhenrik@qbee.io>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+PKG_USE_MIPS16:=0
```

```suggestion
PKG_NAME:=qbee-agent
PKG_VERSION:=2024.23
PKG_RELEASE:=1

PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/qbee-io/qbee-agent.git
PKG_SOURCE_VERSION:=3219c17730eab4c80598494732b4bed4c4c18b51
PKG_MIRROR_HASH:=<needs re-generate>

PKG_LICENSE:=Apache-2.0
PKG_LICENSE_FILES:=LICENSE
PKG_MAINTAINER:=Jon Henrik Bjørnstad <jonhenrik@qbee.io>

PKG_BUILD_DEPENDS:=golang/host
PKG_BUILD_FLAGS:=no-mips16
PKG_BUILD_PARALLEL:=1
```

sort and format variables

remove deprecated `PKG_USE_MIPS16` and useless `PKG_GIT_` stuffs